src/makeRobotAccountant.js

```diff
@@ -6,7 +6,27 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let callCount = 0;
+  let result = [];
+
+  return function getSum(num) {
+    callCount++;
+    result.push(num);
+
+    return function next(nextNum) {
+      result.push(nextNum);
+
+      const sum = result.reduce((acc, curr) => acc + curr);
```

Comment:
Having an array `result` and a reduce method is an overkill for summing two values, right?

Instead inside of `next` function simply add and return `num + nextNum`.

Also I recommend to use arrow functions. You won't need to mess with naming those functions.